README.md

```diff
@@ -0,0 +1,11 @@
+__**CascadeBot**__
+
+CascadeBot runs on JDK 11, which is good because yeet? Apparently I also need to mention the rage that Ryan has thrown at it, not sure who Ryan is though :D
+
+**Credits**
+
+Although we haven't taken code directly from them, the projects listed in LICENSES have provided inspiration, and we thought we'd mention them.
+
+**Sidenote**
+
+Binary told me to be professional in this, although it is the first commit so I thought I'd completely ignore that and have some fun. Something to tell the kids!
```

Comment:
@weeryan17 should I let this pass? 👀

Reply:
You heard the man, make it professional! ;)

Reply:
yessir